cwl/src/main/scala/cwl/CommandLineTool.scala

```diff
@@ -311,10 +311,30 @@ object CommandLineTool {
     case (StringOrInt.Int(_), StringOrInt.String(_)) => true
     // String > Int
     case (StringOrInt.String(_), StringOrInt.Int(_)) => false
+    // 2.13 match may not be exhaustive
+    case oh => throw new Exception(s"Programmer error!: $oh")
   })
 
+  // https://github.com/scala/bug/issues/4097#issuecomment-292388627
+  implicit def IterableSubclass[CC[X] <: Iterable[X], T: Ordering] : Ordering[CC[T]] = {
```

Comment:
With 2.13 Cromwell apparently does not know how to make an `Ordering` for a `List` whereas in 2.12 it does 🤷

Reply:
I think ordering list is a bit arbitrary (maybe that's why they removed it? no idea it existed). what's the criteria for ordering `List`? length?

Reply:
It looks like it's comparing element by element; if one list is a prefix of the other the shorter list is ordered first.